### PR TITLE
[Pages] WS migration revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/pages/src/content/migrations/migrating-from-workers/index.md [hunk 1]
```diff
@@ -6,32 +6,32 @@ pcx-content-type: tutorial
...
+After you have recorded your **build command** and **build directory** in a separate location, remove everything else from your application, and push the new version of your project up to GitHub. Follow the ["Get started" guide](/get-started) to add your project to Cloudflare Pages, using the **build command** and **build directory** that you saved earlier.
 
 If you choose to use a custom domain for your Pages, you can set it to the same custom domain as your currently deployed Workers application. When Pages finishes the initial deploy of your site, you will need to delete the Workers application to start sending requests to Cloudflare Pages.
 
 ## Cleaning up your old application and assigning the domain
 
-Once you've deployed your application, you should go into the [Workers dashboard](https://dash.cloudflare.com/?to=/:account/workers) and remove your old Workers project.
+After you have deployed your application, navigate to [Workers](https://dash.cloudflare.com/?to=/:account/workers) and remove your old Workers project by going to your project > **Settings** > **Delete**.
```
Comment (deadlypants1973) [inline products/pages/src/content/migrations/migrating-from-workers/index.md]: @signalnerve is this the instruction you mean to indicate?

Comment (kristianfreeman) [reply products/pages/src/content/migrations/migrating-from-workers/index.md]: yep, exactly!

File: products/pages/src/content/migrations/migrating-from-workers/index.md [hunk 2]
```diff
@@ -6,32 +6,32 @@ pcx-content-type: tutorial
...
+Cloudflare Pages provides built-in defaults for every aspect of serving your site. Currently, you cannot port custom behavior in your Workers script — such as custom caching logic — over to your Cloudflare Pages. If you need that behavior, keep using your Cloudflare Workers application.
 
-If you're building a straightforward static site, and haven't ever considered adding custom behavior to our Workers Sites templates, moving to Pages is a great fit. You'll get added benefits like [preview deployments](/platform/preview-deployments) and automatic branch deploys with no extra configuration needed.
+If you have not needed to add custom behavior to a Workers Sites template, building a static site with Cloudflare Pages will be easy to do. You will be able to access features like [preview deployments](/platform/preview-deployments) and automatic branch deploys with no extra configuration needed.
```
Comment (lukeed) [inline products/pages/src/content/migrations/migrating-from-workers/index.md]:
```suggestion
If you did not need to customize a Workers Sites template, migrating your static site to Cloudflare Pages will be easy to do. You will be able to access features like [preview deployments](/platform/preview-deployments) and automatic branch deploys with no extra configuration needed.
```

File: products/pages/src/content/migrations/migrating-from-workers/index.md [hunk 3]
```diff
@@ -6,32 +6,32 @@ pcx-content-type: tutorial
...
+After you have recorded your **build command** and **build directory** in a separate location, remove everything else from your application, and push the new version of your project up to GitHub. Follow the ["Get started" guide](/get-started) to add your project to Cloudflare Pages, using the **build command** and **build directory** that you saved earlier.
 
 If you choose to use a custom domain for your Pages, you can set it to the same custom domain as your currently deployed Workers application. When Pages finishes the initial deploy of your site, you will need to delete the Workers application to start sending requests to Cloudflare Pages.
 
 ## Cleaning up your old application and assigning the domain
 
-Once you've deployed your application, you should go into the [Workers dashboard](https://dash.cloudflare.com/?to=/:account/workers) and remove your old Workers project.
+After you have deployed your application, navigate to [Workers](https://dash.cloudflare.com/?to=/:account/workers) and remove your old Workers project by going to your project > **Settings** > **Delete**.
```
Comment (marciocloudflare) [inline products/pages/src/content/migrations/migrating-from-workers/index.md]: Change to `going to your **Project**`

**Review by marciocloudflare — APPROVED** [review]

File: products/pages/src/content/migrations/migrating-from-workers/index.md (hunk 3 above)
Comment (deadlypants1973) [inline products/pages/src/content/migrations/migrating-from-workers/index.md]:
```suggestion
After you have deployed your application, navigate to [Workers](https://dash.cloudflare.com/?to=/:account/workers) and remove your old Workers project by going to your **Workers project** > **Settings** > **Delete**.
```

**Pull request merged**